### PR TITLE
Generation of new descriptor_elabbook files

### PR DESCRIPTION
**Pull request by killianrochet**

### CODE REVIEW

**Comment by JuliaSprenger** [comment]
Hi @killianrochet This PR is based part of your changes in #64, right? So the net changes you want to include via this PR are not the diffs shown here, but https://github.com/killianrochet/DigLabTools/compare/ELAB/APIInterface...killianrochet:DigLabTools:conversionToOdmlTable . For PRs like this it would be good to indicate this dependency in the PR description, to ensure the underlying PR is merged first.

**Comment by JuliaSprenger** [comment]
As a next step in this PR you should make the test more concrete. Right now it only fails if an error is raised in the conversion step.

**Comment by JuliaSprenger** [comment]
Hi @killianrochet This PR still contains a lot of changes already merged into `main`. I think once you continue adding commits here, github should drop the currently cached diffs. Could you try?

File: redcap_bridge/utils.py [hunk 1]
```diff
@@ -295,3 +295,32 @@ def parse_choices(choice_str, annotation_str):
     default_choice_label = re.sub(r'\{.*?\}', '', default_choice_label)
 
     return list(choice_labels), default_choice_label
+
+
+def conversion_to_odml_table_descriptor(full_elabbook_csv, session_number):
```
Comment (JuliaSprenger) [inline redcap_bridge/utils.py:300]:
```suggestion
def conversion_to_odmltables_descriptor(full_elabbook_csv, session_number):
```

**Comment by JuliaSprenger** [comment]
Doesn't look like it... :/ I guess you have to clean this PR manually / copy the actual changes to a new branch. Luckily it's only very few files.

**Comment by killianrochet** [comment]
Yes, I'm sorry for that mistake ... So i'll copy all file and close this PR right ?

**Comment by JuliaSprenger** [comment]
Yes please!

**Comment by JuliaSprenger** [comment]
@killianrochet Alternatively you could try rebasing this branch onto master, but you might run into conflicts in the process potentially...

**Pull request closed without merging**